binutils/plan.sh

```diff
@@ -58,14 +58,6 @@ do_prepare() {
   # path to bash. Until a bash plan is created, we're going to wing this...
   bash=/bin/bash
 
-  # Make `--enable-new-dtags` the default so that the linker sets `RUNPATH`
-  # instead of `RPATH` in ELF binaries. This is important as `RPATH` is
-  # overridden if `LD_LIBRARY_PATH` is set at runtime.
```

Comment:
One interesting thing here is that I think this comment is wrong. At least, the man page says it is wrong.

Reply:
Interestingly, this comment in hab-plan-build makes the same assumption that `RUNPATH`  isn't overridable whereas my reading of the manpage says the opposite:

```
  # Build `$LD_RUN_PATH` containing each path in our own `${pkg_lib_dirs[@]}`
  # array and then each direct run dependency's `LD_RUN_PATH` entry if one
  # exists. This ensures that the resulting `RUNPATH` (like `RPATH`, but not
  # overridable) entries only contain **direct** **runtime** paths. If a
  # dependency's lib directory needs to be set in the resulting `RUNPATH`
  # sections of an ELF binary, it must be a direct dependency, not transitive.
```